tools/adventure-pack/goodies/typescript/Iterator.prototype.every/index.ts

```diff
@@ -0,0 +1,25 @@
+import "../Iterator.prototype.toIterable";
+import { iteratorPrototype } from "../Iterator.prototype";
+
+declare global {
+  interface Iterator<T> {
+    every(
+      this: Iterator<T>,
+      callbackfn: (value: T, index: number) => unknown,
+    ): boolean; 
+  }
+}
+
+iteratorPrototype.every ??= function <T> (
+  this: Iterator<T>,
+  callbackfn: (element: T, index: number) => unknown,
+): boolean {
+  let index = 0;
+  for(const element of this.toIterable()) {
+    if(!callbackfn(element, index)) {
```

Comment:
It looks like Prettier wasn't run on this file. (See the health report as well!)

Reply:
ack!